src/index.html

```diff
@@ -0,0 +1,282 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet" />
+  </head>
+  <body >
+    <nav class="nav">
+      <img class="nav__logo" src="/image/logo.png" alt="" />
+      <div class="nav__menu">
+        <a href="./" class="nav__item">Home</a>
+        <a href="./product.html" class="nav__item">Product</a>
+        <a href="./pricing.html" class="nav__item">Pricing</a>
+        <a href="./about.html" class="nav__item">About Us</a>
+        <a href="./blog.html" class="nav__item">Blog</a>
+        <a href="./contact.html" class="nav__item">Contact</a>
+        <button class="nav__btn">Free trial</button>
+      </div>
+    </nav>
+    <div class="hero">
+      <div
+        class="hero__background"
+      >
+      </div>
+      <h1 class="hero__title">The Best Software to Grow your Sales and Services</h1>
+      <p class="hero__description">
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Fuga, natus
+        voluptatum aspernatur illo earum voluptates quae unde. Cum totam dolorem
+        sint nesciunt molestias fugiat obcaecati doloremque sit!
+      </p>
+      <div class="hero__trial">
+        <input
+          class="hero__input"
+          type="text"
+          placeholder="Enter Your Email"
+          name=""
+          id=""
+        />
+        <button class="hero__btn">Get Free trial</button>
+      </div>
+      <img class="hero__image" src="/image/dashboard.png" alt="" />
+    </div>
+    <div class="quality">
+      <p class="quality__tag ">High quality</p>
+      <h1 class="quality__title">We have the Best Solution for your Business</h1>
+      <div class="quality-section quality-section--light">
+        <div class="quality-section__card">
+          <img class="quality-section__icon" src="/image/Vicon.png" alt="" class="flex-section__img" />
+          <h1 class="quality-section__title">
+            High security to protect from piracy
+          </h1>
+          <p class="quality-section__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+        <div class="quality-card">
+          <img class="quality-card__icon" src="/image/LikeIcon.svg" alt="" class="flex-section__img" />
```

Comment:
Avoid using double-class attributes. the one on the right side should be removed